packages/ai-constructs/src/conversation/conversation_handler_construct.ts

```diff
@@ -67,6 +86,7 @@ export class ConversationHandlerFunction
           // For custom entry we do bundle SDK as we can't control version customer is coding against.
           bundleAwsSDK: !!this.props.entry,
         },
+        loggingFormat: LoggingFormat.JSON,
```

Comment:
To be able to actually see the debug logs, don't we need to also expose the `ApplicationLogLevel` property so customers can configure it? if I recall cdk sets the default as `INFO`?

Reply:
Correct. I called this out in the "Out of scope" section in the PR description.